plugins/rise-lab.js

```diff
@@ -0,0 +1,69 @@
+// Plugin for jupyterlab_rise presentations, based on the generic plugin.
+// Slides are advanced by key presses; the deck is over when the URL stays unchanged.
+
+export const help =
+`Exports a jupyterlab_rise presentation by advancing with space key presses and
+ending when the part after the # in the URL did not change. To use it, start
+"jupyter lab", and call decktape with a URL of the form:
+http://server:port/rise/your_path/your_notebook.ipynb?token=your_token`;
+
+export const create = page => new RISElab(page);
+
+class RISElab {
+  constructor(page) {
+    this.page = page;
+    this.currentSlide = undefined;
+  }
+
+  getName() {
+    return 'RISE-lab';
+  }
+
+  isActive() {
+    return true;
```

Comment:
This can break other plugins activation. Would there be a more robust way to detect it's called in a jupyterlab_rise context?

Alternatively, we could exclude the plugin here: https://github.com/astefanutti/decktape/blob/6735cec96dad6b32ee43d04f858b9794081cb866/decktape.js#L350
but I'd rather go with the former approach.

Reply:
Ah, good point. So this would break automatic plugin selection. I'll check if there is a way to tell whether we are in the correct context without waiting for the presentation to be loaded, but it will take a while until I'll get around to that.